docs/interviewing_options.md

```diff
@@ -1 +1,134 @@
 # Interviewing Options
+
+Overview the most common interviewing options to get a sense of how they are
+typically used, what is: involved, pros, cons, and biases.
+
+
+
+## Whiteboarding
+
+Whiteboarding can be the use a of a physical whiteboard or a online code/sharing
+editor where a candidate is given a problem and expected to come up with a
+solution within a specific timeframe, and normally think aloud their problem.
+
+Typically the problems candidates are given are algorithmic or puzzle type
+questions typically of what you mind find on [leetcode](https://leetcode.com) or [hackerrank](https://www.hackerrank.com/)
+
+#### Pros
+
+- identifying candidates whom are exceptionally brilliant, quick thinkers, and
+  extremely knowledgable about the specific technical topic at hand
+- See how well candidates can perform under pressure and scrutiny in real time.
+- If done online can help companies and candidates optimize on costs and time
+  of being on-site and in-person
+- Generally can be a low effort (cheap) way to screen out candidates whom might
+  not be technical enough for the role at hand. This can be effect in a company
+  where there are many many applicants.
+
+#### Cons
+
+- Can lead to a lot of false negatives
+- Can bias against people whom have family commitments (caregivers,
```

Comment:
Change to something like... 

Can bias against people who have caregiver commitments (parents, those taking care of a loved one, etc.)

Why change? Because sometimes we care for individuals who are not technically family, but we view them as family.

Reply:
How is a white board bias against caretakers?

Reply:
I'll remove the bias part for white boarding. You are right, it is no different than a regular interview. There might be some with the amount of preparation needed to practice in some of these sorts of interviews.